src/site/content/en/metrics/lcp/index.md

```diff
@@ -1,22 +1,26 @@
 ---
-title: Largest Contentful Paint
-subhead: Making it easier to know when a page's important content has loaded.
+layout: post
+title: Largest Contentful Paint (LCP)
 authors:
   - philipwalton
 date: 2019-08-08
-hero: hero.jpg
-# You can adjust the position of your hero image with this property.
-# Values: top | bottom | center (default)
-hero_position: top
-alt: Artist painting tools
+updated: 2019-11-07
 description: |
-  Use the Largest Contentful Paint API to optimize for faster page loads.
+  This post introduces the Largest Contentful Paint (LCP) metric and explains
+  how to measure it
 tags:
-  - post # post is a required tag for the article to show up in the blog.
   - performance
   - metrics
 ---
 
+{% Aside %}
+  Largest Contentful Paint (LCP) is an important, user-centric metric for
+  measuring [perceived load speed](/user-centric-metrics/#types-of-metrics)
+  because it marks the point in the page load timeline when the page's main
+  content has likely loaded&mdash;a fast LCP helps reassure the user that the
+  page is [useful](/user-centric-metric/#user-centric-metric-questions).
+{% endAside %}
+
```

Comment:
Couldn't figure out how to comment in the appropriate line but it seems to me like the link in line 32 must be updated.

Reply:
Yeah, this has been fixed in a subsequent commit.